internal/support/logging/router_test.go

```diff
@@ -92,7 +104,7 @@ func TestAddLog(t *testing.T) {
 			http.StatusBadRequest},
 	}
 	// create test server with handler
-	ts := httptest.NewServer(LoadRestRoutes())
+	ts := httptest.NewServer(LoadRestRoutes(mockDIC()))
```

Comment:
See my other comment regarding simplifying this test to target the REST handler rather than loading all the routes and making a mock HTTP request.

Reply:
See above.